build.gradle

```diff
@@ -32,8 +32,9 @@ swaggerSources {
         inputFile = file("$buildDir/resources/main/api.yaml")
         code {
             language = 'spring'
+            library = 'spring-mvc'
             configFile = file("$buildDir/resources/main/config.json")
-            dependsOn validation
+//            dependsOn validation
```

Comment:
@kjarosh @szbrnl Tutaj się dzieją RZECZY - walidacja się wywala z dowolnym testowanym przeze mnie wariantem `- in: query`, chociaż generowanie działa i https://editor.swagger.io/ nic nie narzeka, także nie bardzo ogarniam, co mam zrobić, żeby walidacja też była szczęśliwa (coś z wersjami? coś ma być inaczej? nie wiem?)

Reply:
Ooo, ten enum mu nie pasuje - z `integer` przechodzi

Reply:
Em, nawet samo
```      
parameters:
    - in: query
      name: sprintState
      required: true
      type: array
      items:
     type: string
     enum: [ala, ma, kota]
```
go uszczęśliwia, tylko odwołanie `enum: *SPRINT_STATE` już nie (chociaż to działa po zakomentowaniu tej walidacji)

Reply:
To ten, ja to umiem rozwiązać poprzez pozbycie się linijki `dependsOn validation`

Reply:
Ta śmieszna gwiazdka jest stąd jakby co:
https://swagger.io/docs/specification/2-0/enums/ --> Reusable Enums
My chyba właśnie używamy wersji 2.0?